package legym.fuck.widget.exceptions;

import java.lang.System;

/**
 * @author ldh
 * 时间: 2022/3/23 22:51
 */
@kotlin.Metadata(mv = {1, 6, 0}, k = 1, d1 = {"\u0000\u0012\n\u0002\u0018\u0002\n\u0002\u0018\u0002\n\u0000\n\u0002\u0010\u000e\n\u0002\b\u0002\u0018\u00002\u00020\u0001B\u000f\u0012\b\u0010\u0002\u001a\u0004\u0018\u00010\u0003\u00a2\u0006\u0002\u0010\u0004\u00a8\u0006\u0005"}, d2 = {"Llegym/fuck/widget/exceptions/MaliciousUserException;", "Ljava/lang/Exception;", "message", "", "(Ljava/lang/String;)V", "app_debug"})
public final class MaliciousUserException extends java.lang.Exception {
    
    public MaliciousUserException(@org.jetbrains.annotations.Nullable()
    java.lang.String message) {
        super();
    }
}